x/gov/abci.go

```diff
@@ -5,7 +5,9 @@ import (
 	"time"
 
 	"cosmossdk.io/collections"
+	"google.golang.org/protobuf/proto"
```

Comment:
How is this code compiling? Apparently proto is imported but not used, please delete it.